### DISCUSSION TITLE
Error: unable to access 'scoop.git' SSL certificate problem

### DISCUSSION THREAD

**Discussion by maulikp66** [ScoopInstaller/Scoop#5639] [Help and Questions]
Getting this error when installing scoop.

Steps:

1. downloaded install.ps1 file into downloads folder
2. cd into downloads
3. run `Set-ExecutionPolicy RemoteSigned -scope CurrentUser`
4. then run `Invoke-Expression ./install.ps1`
5.  Error:  unable to access 'https://github.com/ScoopInstaller/Scoop.git/': SSL certificate problem: self signed certificate in certificate chain

Please advice on how to fix?
Thanks

**Comment by rashil2000** [comment]
What happens when you run the command `git clone https://github.com/ScoopInstaller/Scoop.git` in the terminal?
